Examples/Python/src/Base.cpp

```diff
@@ -238,23 +238,29 @@ void addAlgebra(Acts::Python::Context& ctx) {
         Acts::Vector2 v;
         v << a[0], a[1];
         return v;
-      }));
+      }))
+      .def("__getitem__",
+           [](const Acts::Vector2& self, Eigen::Index i) { return self[i]; });
 
   py::class_<Acts::Vector3>(m, "Vector3")
       .def(py::init<double, double, double>())
       .def(py::init([](std::array<double, 3> a) {
         Acts::Vector3 v;
         v << a[0], a[1], a[2];
         return v;
-      }));
+      }))
+      .def("__getitem__",
+           [](const Acts::Vector3& self, Eigen::Index i) { return self[i]; });
 
   py::class_<Acts::Vector4>(m, "Vector4")
       .def(py::init<double, double, double, double>())
       .def(py::init([](std::array<double, 4> a) {
         Acts::Vector4 v;
         v << a[0], a[1], a[2], a[3];
         return v;
-      }));
+      }))
+      .def("__getitem__",
+           [](const Acts::Vector4& self, Eigen::Index i) { return self[i]; });
```

Comment:
I might have missed it but what is the context of those changes ?

Reply:
Without the `__getitem__` functions it is not possible to access the content of an `Acts::Vector3` in python, and thus would not be possible to compute the radius to set the extra id.

Reply:
I added them also to the other vectors for consistency

Reply:
Make sense !